### DISCUSSION TITLE
"0 Assignments Copied"

### DISCUSSION THREAD

**Discussion by BabyBlue2194** [JamesNZL/notion-assignment-import#255] [Q&A] [answered]
I got the add-on to work on Chrome. But it will not export to notion. it just says "Created 0 assignments" after the course is successfully copied. 
Any suggestions?

**Comment by JamesNZL** [comment, marked as answer]
Do these assignments have due dates? Are these due dates in the future?

**Reply by BabyBlue2194** [reply]
After looking through the discord Ive realized that probably is the problem. When the new quarter starts for me Ill try it again. :)
It only works on Chrome for me right now. Firefox doesnt give me the availability to copy anything  ill play around with permissions, tht may be the problem.

**Reply by JamesNZL** [reply]
Yeah, one of Firefox's recent updates has broken the extension after an extension update—my extension displays an alert when the extension updates, and it seems that Firefox is no longer able to display the alert (so the extension 'freezes' because there's actually an alert box open, but you can't dismiss the alert because it's invisible)

As a workaround, you can:

1. Go to the `Options` page from the meatball menu
2. Change the URL to `.../popup/popup.html`
3. You will see the alert popup—click `OK`
4. The extension popup should function correctly now from the top toolbar
